extension/src/background/messages/oauth.ts

```diff
@@ -48,8 +51,29 @@ async function getUserDataByUsername(
   return null;
 }
 
-async function getTwitchUserByAccessToken(
+async function getUserChatColor(
   accessToken: string,
+  userId: number
+): Promise<string | null> {
+  const usernameColor = await fetch(
+    `${TWITCH_API_URL}/chat/color?user_id=${userId}`,
+    {
+      headers: {
+        "Client-ID": CLIENT_ID,
+        Authorization: `Bearer ${accessToken}`,
+      },
+    }
+  );
+
+  if (usernameColor.ok) {
+    const response: ColorChatUser = await usernameColor.json();
+    return response.data[0].color;
+  }
+  return null;
```

Comment:
Just in case: can you set a defaut color to gray? Bcuz with that we can return only a type (which is string).